wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/WorkflowOutputDeclarationElement.scala

```diff
@@ -0,0 +1,16 @@
+package wdl.model.draft3.elements
+
+import common.validation.ErrorOr.ErrorOr
+import simulacrum.typeclass
+
+case class WorkflowOutputDeclarationElement(name: String,
+                                            valueType: String,
+                                            value: String
+                                ) extends LanguageElement {
+  override def children: Seq[String] = Seq(name, valueType, value)
+}
+
+@typeclass
+trait WorkflowOutputDeclarationElementMaker[A] {
```

Comment:
This is why the build has failed: to add this `@typeclass` annotation you need to add this line to the top of the file too (because it's doing implicit conversions behind the scenes and scala makes you declare that in your imports because they're a so-called 'advanced language feature'):
```
import scala.language.ImplicitConversion
```

But... I would actually just remove the trait altogether. I've been thinking of deleting them from the existing classes too since I've been moving towards the more general purpose (and more composable) `FromAtoB[A, B]` trait for the conversions 

Eg `object Draft3FileElementFromAst extends FromAst[FileElement]` where `FromAst[FileElement]` extends `FromAtoB[Ast, FileElement]`).

Reply:
Should I go ahead and remove the typeclass trait from WorkflowDefinitionElement as well?